@xen-orchestra/web-core/lib/components/tab/VtsTabList.vue

```diff
@@ -1,6 +1,6 @@
-<!-- v1.0 -->
+<!-- v2 -->
```

Comment:
Based on the `Tabs bar changelog`, the `v2` implement the `scroll bar option`

Reply:
In the Vue component, the scrollbar is automatically visible when there is not enough space. I'm not sure if we should have a prop to always enable it.